appeals/web/src/server/appeals/appeal-details/lpa-questionaire/lpa-questionnaire.controller.js

```diff
@@ -35,7 +35,7 @@ const renderLpaQuestionnaire = async (request, response, errors = null) => {
 
 	if (lpaQuestionnaire && appealDetails) {
 		const currentUrl = request.originalUrl;
-		const pageComponents = lpaQuestionnairePage(
+		const pageComponents = await lpaQuestionnairePage(
```

Comment:
Was there an instance where it didn't work without the await? I didn't have that problem. Make sense though!

Reply:
I had to add this because `initialiseAndMapAppealData` in `appeal.mapper.js` now needs to hit the `getUserByRoleAndId` service method to get the assigned case officer and inspector details from AD. As this is an async call, I had to change the function and those in its chain of callers to `async`, hence the `await` being added here